tox.ini

```diff
@@ -208,10 +212,8 @@ commands = {envbindir}/sphinx-build -v -W -d {toxinidir}/docs/_build/doctrees -b
 [testenv:codecov]
 basepython = python{env:TOXPYTHON:{env:TRAVIS_PYTHON_VERSION:3.8}}
 passenv = TOXENV CI TRAVIS TRAVIS_* CODECOV_*
-deps = codecov>=1.4.0
-# commands = codecov -e TOXENV
-# Workaround for https://github.com/codecov/codecov-python/issues/158
-commands = {envpython} run_codecov.py -e TOXENV
```

Comment:
This is awesome if it's really fixed.

Reply:
yeah.. travis-ci has gone to shit and will soon be charging us, so there are other files to write now, for github actions, i suppose

Reply:
It's been on my todo list to start looking at github actions for the same reason. Haven't gotten around to it yet.